### PR TITLE
Revise signals help text and enums.

### PR DESCRIPTION
**Pull request by jrobbins**
Is this the right way to resolve issue #833?
I can't view the google doc history so I don't know exactly what change was made as a result of the discussion.

Rather than repeat the instructions four times, I added it to the field that introduces that section.  Let me know if you think something else is better.

### CODE REVIEW

File: guideforms.py [hunk 1]
```diff
@@ -117,7 +117,11 @@
          'our <a target="_blank" href="'
          'https://docs.google.com/document/d/'
          '1RC-pBBvsazYfCNNUSkPqAVpSpNJ96U8trhNkfV0v9fk/edit">'
-         'principles of web compatibility</a>.')),
+         'principles of web compatibility</a>.<br>'
+         '<br>'
+         'Please include citation links below where possible. Examples include '
+         'resolutions from relevant standards bodies (e.g. W3C Working Group), '
```
Comment (jpmedley) [inline guideforms.py:123]:
```suggestion
         'resolutions from relevant standards bodies (e.g. W3C working group), '
```

File: models.py [hunk 2]
```diff
@@ -1076,7 +1076,7 @@ class FeatureForm(forms.Form):
...
+      help_text='Describe the degree of <a target="_blank" href="https://sites.google.com/a/chromium.org/dev/blink?pli=1#TOC-Policy-for-shipping-and-removing-web-platform-API-features">interoperability risk</a>. For a new feature, the main risk is that it fails to become an interoperable part of the web platform if other browsers do not implement it. For a removal, please review our <a target="_blank" href="https://docs.google.com/document/d/1RC-pBBvsazYfCNNUSkPqAVpSpNJ96U8trhNkfV0v9fk/edit">principles of web compatibility</a>.<br><br>Please include citation links below where possible. Examples include resolutions from relevant standards bodies (e.g. W3C Working Group), tracking bugs, or links to online conversations.')
```
Comment (jpmedley) [inline models.py:1079]:
```suggestion
      help_text='Describe the degree of <a target="_blank" href="https://sites.google.com/a/chromium.org/dev/blink?pli=1#TOC-Policy-for-shipping-and-removing-web-platform-API-features">interoperability risk</a>. For a new feature, the main risk is that it fails to become an interoperable part of the web platform if other browsers do not implement it. For a removal, please review our <a target="_blank" href="https://docs.google.com/document/d/1RC-pBBvsazYfCNNUSkPqAVpSpNJ96U8trhNkfV0v9fk/edit">principles of web compatibility</a>.<br><br>Please include citation links below where possible. Examples include resolutions from relevant standards bodies (e.g. W3C working group), tracking bugs, or links to online conversations.')
```

File: guideforms.py (hunk 1 above)
Comment (jpmedley) [reply guideforms.py:123]: 'Working group' is not a proper noun if you're not referencing the name of a specific working group.

Comment (jrobbins) [reply guideforms.py:123]: Done.

**Comment by jrobbins** [comment]
Friendly ping for code review.

**Comment by jstenback** [comment]
The changes to the template document were to remove the following:

**Edge: Shipped/In development/Public support/No signals/Negative signals/Opposed
Firefox: Shipped/In development/Public support/No signals/Negative signals/Opposed
Safari: Shipped/In development/Public support/No signals/Negative signals/Opposed**

and replace it with this:

**Signals from other implementations (Gecko, WebKit): fill in according to instructions in [this document](https://docs.google.com/document/d/1xkHRXnFS8GDqZi7E0SSbR3a7CZsGScdxPUWBsNgo-oo/edit#heading=h.tgzhprxcmw4u).**

So I think we'll need a bit more than what's in this PR :( Let me know if those documents are not reachable or what not.

**Comment by jrobbins** [comment]
So, I am thinking that this is basically the following:
+ rename firefox signals to gecko signals
+ rename safari signals to webkit signals
+ remove the fields for edge signals
+ reword some of the existing option values, and also add all the Gecko
custom field value and offer those only for Gecko

I'll update the CL today.

**Comment by jrobbins** [comment]
After reading that doc again, I decided to keep the names for the vendors as Firefox, Edge, and Safari in our UI, and use the names Gecko and WebKit only in the template to generate the draft intent email.

Let me know if this is on the right track.  If so, I will add unit tests and also write new logic to assign a color to the vendor views icons in the UI (the old logic assumed a linear red-to-green meaning based on the enum int value, whereas new ones are now higher enum values).

**Pull request merged**